examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -14017,6 +14069,218 @@
             }
           ]
         },
+        {
+          "name": "Laundry Washer Controls",
+          "code": 83,
+          "mfgCode": null,
+          "define": "LAUNDRY_WASHER_CONTROLS_CLUSTER",
+          "side": "client",
+          "enabled": 0,
+          "attributes": [
+            {
+              "name": "FeatureMap",
+              "code": 65532,
+              "mfgCode": null,
+              "side": "client",
+              "type": "bitmap32",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "0",
+              "reportable": 1,
+              "minInterval": 1,
+              "maxInterval": 65534,
+              "reportableChange": 0
+            },
+            {
+              "name": "ClusterRevision",
+              "code": 65533,
+              "mfgCode": null,
+              "side": "client",
+              "type": "int16u",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "1",
+              "reportable": 1,
+              "minInterval": 1,
+              "maxInterval": 65534,
+              "reportableChange": 0
+            }
+          ]
+        },
+        {
+          "name": "Laundry Washer Controls",
+          "code": 83,
+          "mfgCode": null,
+          "define": "LAUNDRY_WASHER_CONTROLS_CLUSTER",
+          "side": "server",
+          "enabled": 1,
+          "attributes": [
+            {
+              "name": "SpinSpeeds",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "type": "array",
+              "included": 0,
```

Comment:
So these are all turned off because the feature map is 0, so this cluster supports nothing, right?

Reply:
Turned them all on, sorry about that and thank you for pointing it out!